lib/components/popover.vue

```diff
@@ -165,7 +173,12 @@
                     attachment: this.placementParameters.attachment,
                     targetAttachment: this.placementParameters.targetAttachment
                 };
+            },
+
+            setPopoverStyle() {
```

Comment:
Absolutely not needed at all. Currently in the habit of using setters instead of the vars directly, because of the project I'm working on, didn't think about it.